src/biogeochem/CNPhenologyMod.F90

```diff
@@ -166,6 +209,45 @@ subroutine CNPhenologyReadNML( NLFilename )
     
   end subroutine CNPhenologyReadNML
 
+  !-----------------------------------------------------------------------
+  subroutine CNPhenologySetNML( input_onset_thresh_depends_on_veg, input_critical_daylight_method )
+    !
+    ! !DESCRIPTION:
+    ! Set the namelist items for unit-testing
+    !
+    logical, intent(in) :: input_onset_thresh_depends_on_veg
+    integer, intent(in) :: input_critical_daylight_method
+
+    onset_thresh_depends_on_veg = input_onset_thresh_depends_on_veg
+    critical_daylight_method = input_critical_daylight_method
+    if ( (critical_daylight_method < critical_daylight_constant) .or. &
+         (critical_daylight_method > critical_daylight_depends_on_latnveg) )then
+       call endrun(msg="ERROR critical_daylight_method "//errmsg(sourcefile, __LINE__))
+    end if
+  end subroutine CNPhenologySetNML
+  
+  !-----------------------------------------------------------------------
+  subroutine CNPhenologySetParams( )
+    !
+    ! !DESCRIPTION:
+    ! Set the parameters for unit-testing
+    !
```

Comment:
Should the following have unit definitions, e.g., seconds, etc?

Reply:
Adding units. Because these are only used for unit-testing they actually should be pretty static.